nomad.sysext/create.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# vim: et ts=2 syn=bash
+#
+
+RELOAD_SERVICES_ON_MERGE="true"
+
+# Fetch and print a list of available versions.
+# Called by 'bakery.sh list <sysext>.
+function list_available_versions() {
+  curl -fsSL --retry-delay 1 --retry 60 \
+    --retry-connrefused --retry-max-time 60 --connect-timeout 20 \
+    https://api.releases.hashicorp.com/v1/releases/nomad \
+  | jq -r '.[].version' \
+  | sort -Vr
```

Comment:
This will also include build flavours like
1.10.1+ent.musl
1.10.1+ent
1.10.1
1.9.9+ent.musl
1.9.9+ent
1.9.8+ent.musl
1.9.8+ent
 in the list. 

I think we should filter these out.